examples/components/README.md

```diff
@@ -0,0 +1,32 @@
+# Components
+
+## Get started
+
+Clone the repository
+
+```
+$ git@github.com:FirefoxUX/photon-extension-kit.git
+```
+
+Install `web-ext` globally
+
+```
+$ npm install -g web-ext
+```
+
+Navigate to `/photon-extension-kit/examples/components` and run `web-ext`
+
+```
+$ cd photon-extension-kit/examples/components
+$ web-ext run
+```
+
+## Preview
+
+- Download the [repo](https://github.com/FirefoxUX/photon-extension-kit/archive/master.zip) and unzip it
```

Comment:
For now it's _photon.zip_ not master.zip

Reply:
hey @eiro10 it's intended from when we are going to merge into master, thanks anyway!